pkg/services/alerting/notifiers/line.go

```diff
@@ -75,6 +75,11 @@ func (this *LineNotifier) createAlert(evalContext *alerting.EvalContext) error {
 	body := fmt.Sprintf("%s - %s\n%s", evalContext.Rule.Name, ruleUrl, evalContext.Rule.Message)
 	form.Add("message", body)
 
+        if evalContext.ImagePublicUrl != "" {
+        form.Add("imageThumbnail", evalContext.ImagePublicUrl)
```

Comment:
LINE will automatically compress the image coming to their services.
imageThumbnail usually used for the avatar in LINE game.
is there any "bundle" image resizer in grafana ?
or for the sake of simplicity, should i delete the imageThumbnail ?
so we can focus on imageFullsize ?